paint/PAINT_ColourPicker.cpp

```diff
@@ -77,34 +109,74 @@ void ColourPicker::setPrimarySecondaryFromActiveDraw() const
 	}
 }
 
+
+void ColourPicker::swapActiveColour() const
+{
+	primaryColourDisplay_->swapIsActive();
+	secondaryColourDisplay_->swapIsActive();
+
+	setDrawColourFromActive();
+	//update();
+
+	//rerender();
+}
+
+void ColourPicker::updateColourDisplaysFromDrawWindow() const
+{
+	const auto primaryRGBA = drawWindowPtr_->getPrimaryRGBA();
+	const auto secondaryRGBA = drawWindowPtr_->getSecondaryRGBA();
+	if (!swappedDisplays_) {
+		//primaryColourDisplay_->setColour(drawWindowPtr_->getPrimaryColour());
```

Comment:
Try not to leave redundant commented-out code in. Unless it's for future reference, in which case add a comment to indicate that
**MUST**

Reply:
done, removed commented-out lines